content/en/about/feature-stages/index.md

```diff
@@ -16,17 +16,29 @@ within the project, not to the project as a whole. Here is a high level descript
+| Reliability           | The user should not expect the feature to be reliable may be untested.                                                                                                                                                                                                                                                                                                                                                                                                     | Because the feature is relatively new, and may lack complete end-to-end tests, enabling the feature via a flag might expose bugs which destabilize Istio (e.g. a bug in a control loop might rapidly create excessive numbers of objects, exhausting API storage).                                                                                                                                                                                 | Because the feature has `e2e` tests, enabling the feature should not create new bugs in unrelated features. Because the feature is relatively new, it may have minor bugs.                                                                                                                                                                                                                                                                                                                  | High. The feature is well tested and stable and reliable for all uses.                                                                     |
```

Comment:
Same comment for e2e. Why do we say `High` in the stable column but nothing in the others?